nautobot/ipam/tables.py

```diff
@@ -688,7 +688,6 @@ class Meta(BaseTable.Meta):
 class VLANTable(StatusTableMixin, RoleTableMixin, BaseTable):
     pk = ToggleColumn()
     vid = tables.TemplateColumn(template_code=VLAN_LINK, verbose_name="ID")
-    location = tables.Column(linkify=True)
```

Comment:
As mentioned above, should probably add a `locations` column as a `LinkedCountColumn` here and below.